webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js

```diff
@@ -16,5 +19,10 @@ export const UPDATE_CLICKED = 'UPDATE_CLICKED';
 
 export const TASKS_SEARCH_PROPS = {
   ...getControllerSearchProps('tasks'),
-  controller: 'foreman_tasks/tasks'
+  controller: 'foreman_tasks/tasks',
 };
+
+export const INFO = 'info';
+export const SUCCESS = 'success';
+export const ERROR = 'error';
+export const WARNING = 'warning';
```

Comment:
can you move them to `/Components/common/`?
and maybe store all of them as `TOASTS_TYPES`.

Reply:
Great suggestion, changed